installer/templates/phx_single/lib/app_name_web.ex

```diff
@@ -100,6 +100,7 @@ defmodule <%= @web_namespace %> do
     quote do<%= if @html do %>
       # Import basic HTML rendering capabilities (tags, forms, etc)
       use Phoenix.HTML
+      use Phoenix.Component
```

Comment:
```suggestion
      import Phoenix.Component
```

Reply:
This should not be necessary because view/live_view/live_component all already `use Phoenix.Component`.

Reply:
```suggestion
```